### ISSUE TITLE
Flat review doesn't work in Magic DVD Ripper

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2911]
**Reported by jkinnunen on 2013-01-05 06:30**
It is not possible to rip DVDs with Magic DVD Ripper, since NVDA's flat review doesn't work in the program at all. When pressing NVDA+Numpad7 while the program is open, NVDA reports "No flat review for this object".

A trial version of Magic DVD Ripper can be downloaded at http://www.magicdvdripper.com/.

**Comment by leonardder** [comment]
@jkinnunen: Do you still consider this issue to be relevant?

**Comment by Adriani90** [comment]
@jkinnunen can you still reproduce this in last version of magic DVD ripper?

**Comment by Adriani90** [comment]
We do not have an answer from the initial author for a very long time. If someone still has this problem in NVDA 2019.1.1 and the last version of DVD Ripper, please comment on it and we can reopen the discussion.
